Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/survey/server/DataDonationApiV1.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.datadonation.survey.server
+
+import com.google.gson.annotations.SerializedName
+import de.rki.coronawarnapp.server.protocols.internal.ppdd.EdusOtp
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface DataDonationApiV1 {
+
+    data class DataDonationResponse(
+        @SerializedName("expirationDate") val expirationDate: String?
```

Comment:
Can this be null? Or is it being `null` an error case?

Reply:
obsolete, since expiration date is not used anymore